2023-12-25 07:31:37,269 INFO    StreamThr :27488 [internal.py:wandb_internal():87] W&B internal server running at pid: 27488, started at: 2023-12-25 07:31:37.269431
2023-12-25 07:31:37,283 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: status
2023-12-25 07:31:37,284 INFO    WriterThread:27488 [datastore.py:open_for_write():85] open: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\run-oavr6n5s.wandb
2023-12-25 07:31:37,285 DEBUG   SenderThread:27488 [sender.py:send():336] send: header
2023-12-25 07:31:37,285 DEBUG   SenderThread:27488 [sender.py:send():336] send: run
2023-12-25 07:31:37,778 INFO    SenderThread:27488 [dir_watcher.py:__init__():216] watching files in: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files
2023-12-25 07:31:37,778 INFO    SenderThread:27488 [sender.py:_start_run_threads():1067] run started: oavr6n5s with start time 1703457097.269431
2023-12-25 07:31:37,778 DEBUG   SenderThread:27488 [sender.py:send_request():363] send_request: summary_record
2023-12-25 07:31:37,778 INFO    SenderThread:27488 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-12-25 07:31:37,778 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: check_version
2023-12-25 07:31:37,778 DEBUG   SenderThread:27488 [sender.py:send_request():363] send_request: check_version
2023-12-25 07:31:37,841 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: run_start
2023-12-25 07:31:37,872 DEBUG   HandlerThread:27488 [system_info.py:__init__():31] System info init
2023-12-25 07:31:37,872 DEBUG   HandlerThread:27488 [system_info.py:__init__():46] System info init done
2023-12-25 07:31:37,872 INFO    HandlerThread:27488 [system_monitor.py:start():151] Starting system monitor
2023-12-25 07:31:37,872 INFO    SystemMonitor:27488 [system_monitor.py:_start():116] Starting system asset monitoring threads
2023-12-25 07:31:37,873 INFO    HandlerThread:27488 [system_monitor.py:probe():172] Collecting system info
2023-12-25 07:31:37,874 INFO    SystemMonitor:27488 [interfaces.py:start():168] Started cpu
2023-12-25 07:31:37,874 INFO    SystemMonitor:27488 [interfaces.py:start():168] Started disk
2023-12-25 07:31:37,874 INFO    SystemMonitor:27488 [interfaces.py:start():168] Started gpu
2023-12-25 07:31:37,874 INFO    SystemMonitor:27488 [interfaces.py:start():168] Started memory
2023-12-25 07:31:37,898 INFO    SystemMonitor:27488 [interfaces.py:start():168] Started network
2023-12-25 07:31:37,898 DEBUG   HandlerThread:27488 [system_info.py:probe():195] Probing system
2023-12-25 07:31:37,899 DEBUG   HandlerThread:27488 [git.py:repo():40] git repository is invalid
2023-12-25 07:31:37,899 DEBUG   HandlerThread:27488 [system_info.py:probe():241] Probing system done
2023-12-25 07:31:37,900 DEBUG   HandlerThread:27488 [system_monitor.py:probe():181] {'os': 'Windows-10-10.0.19045-SP0', 'python': '3.9.16', 'heartbeatAt': '2023-12-24T22:31:37.898071', 'startedAt': '2023-12-24T22:31:37.264547', 'docker': None, 'cuda': None, 'args': ('--task', 'compatibility', '--train_batch', '48', '--valid_batch', '64', '--n_epochs', '4', '--learning_rate', '1e-5', '--save_dir', 'F:\\Projects\\outfit-transformer\\model\\saved_model', '--work_dir', 'F:/Projects/outfit-transformer', '--data_dir', 'F:/Projects/datasets/polyvore_outfits', '--load_model_name', 'checkpoint_CP_2023-12-25_0_0.778'), 'state': 'running', 'program': 'F:\\Projects\\outfit-transformer\\train.py', 'codePath': 'train.py', 'host': 'DESKTOP-0JN8HPO', 'username': 'WONJUN', 'executable': 'C:\\Users\\owj04\\anaconda3\\envs\\AIKU\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3593.0, 'min': 0.0, 'max': 3593.0}, 'cpu_freq_per_core': [{'current': 3593.0, 'min': 0.0, 'max': 3593.0}], 'disk': {'total': 488.28026962280273, 'used': 236.38006973266602}, 'gpu': 'NVIDIA GeForce RTX 2060', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 2060', 'memory_total': 6442450944}], 'memory': {'total': 15.932670593261719}}
2023-12-25 07:31:37,900 INFO    HandlerThread:27488 [system_monitor.py:probe():182] Finished collecting system info
2023-12-25 07:31:37,900 INFO    HandlerThread:27488 [system_monitor.py:probe():185] Publishing system info
2023-12-25 07:31:37,900 DEBUG   HandlerThread:27488 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-12-25 07:31:37,900 DEBUG   HandlerThread:27488 [system_info.py:_save_pip():67] Saving pip packages done
2023-12-25 07:31:37,901 DEBUG   HandlerThread:27488 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-12-25 07:31:37,902 ERROR   HandlerThread:27488 [system_info.py:_save_conda():85] Error saving conda packages: [WinError 2] 지정된 파일을 찾을 수 없습니다
Traceback (most recent call last):
  File "C:\Users\owj04\anaconda3\envs\AIKU\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 81, in _save_conda
    subprocess.call(
  File "C:\Users\owj04\anaconda3\envs\AIKU\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\owj04\anaconda3\envs\AIKU\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\owj04\anaconda3\envs\AIKU\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] 지정된 파일을 찾을 수 없습니다
2023-12-25 07:31:37,902 DEBUG   HandlerThread:27488 [system_info.py:_save_conda():86] Saving conda packages done
2023-12-25 07:31:37,902 INFO    HandlerThread:27488 [system_monitor.py:probe():187] Finished publishing system info
2023-12-25 07:31:37,918 DEBUG   SenderThread:27488 [sender.py:send():336] send: files
2023-12-25 07:31:37,918 INFO    SenderThread:27488 [sender.py:_save_file():1321] saving file wandb-metadata.json with policy now
2023-12-25 07:31:37,935 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: stop_status
2023-12-25 07:31:37,935 DEBUG   SenderThread:27488 [sender.py:send_request():363] send_request: stop_status
2023-12-25 07:31:38,252 DEBUG   SenderThread:27488 [sender.py:send():336] send: telemetry
2023-12-25 07:31:38,560 INFO    wandb-upload_0:27488 [upload_job.py:push():138] Uploaded file C:\Users\owj04\AppData\Local\Temp\tmp96e7py6mwandb\lq68xvop-wandb-metadata.json
2023-12-25 07:31:38,794 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_created():275] file/dir created: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\wandb-summary.json
2023-12-25 07:31:38,794 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_created():275] file/dir created: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\requirements.txt
2023-12-25 07:31:38,794 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_created():275] file/dir created: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\wandb-metadata.json
2023-12-25 07:31:38,794 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_created():275] file/dir created: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\conda-environment.yaml
2023-12-25 07:31:42,286 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: status_report
2023-12-25 07:31:43,826 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_created():275] file/dir created: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\output.log
2023-12-25 07:31:45,841 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_modified():292] file/dir modified: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\output.log
2023-12-25 07:31:47,857 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_modified():292] file/dir modified: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\output.log
2023-12-25 07:31:48,076 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: status_report
2023-12-25 07:31:48,873 INFO    Thread-17 :27488 [dir_watcher.py:_on_file_modified():292] file/dir modified: F:\Projects\outfit-transformer\wandb\run-20231225_073137-oavr6n5s\files\output.log
2023-12-25 07:31:52,951 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: keepalive
2023-12-25 07:31:52,951 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: keepalive
2023-12-25 07:31:53,107 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: status_report
2023-12-25 07:31:57,951 ERROR   memory    :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:57,951 ERROR   cpu       :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:57,951 ERROR   memory    :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:57,951 ERROR   cpu       :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:58,138 DEBUG   HandlerThread:27488 [handler.py:handle_request():144] handle_request: status_report
2023-12-25 07:31:59,961 ERROR   cpu       :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:59,961 ERROR   memory    :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:59,961 ERROR   cpu       :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:31:59,961 ERROR   memory    :27488 [interfaces.py:monitor():129] Failed to sample metric: process no longer exists (pid=11032)
2023-12-25 07:32:00,008 INFO    MainThread:27488 [internal.py:handle_exit():77] Internal process exited
